recipes/tree-gen/all/conanfile.py

```diff
@@ -38,19 +34,33 @@ def _should_build_test(self):
         return not self.conf.get("tools.build:skip_test", default=True, check_type=bool)
 
     @property
-    def _min_cpp_std(self):
+    def _min_cppstd(self):
```

Comment:
Copied from Templates (https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates/cmake_package/all) the adjusts, that's why some methods names and method position were change. The position reflects the invocation order when running Conan.